cmd/agent/main.go

```diff
@@ -93,12 +93,18 @@ func (m *Metrics) GetMetrics() []Element {
 }
 
 func SendMetric(val string) {
-	resp, err := http.Post(val, "text/plain", nil)
-	if err != nil {
-		fmt.Println(err)
-		return
+
+	var err error
+	for _, i := range [3]int{1, 3, 5} {
+		resp, err := http.Post(val, "text/plain", nil)
+		if err == nil {
+			defer resp.Body.Close()
+			return
+		}
+		time.Sleep(time.Duration(i) * time.Second)
+		fmt.Printf("Retry send metric: %d\n", i)
```

Comment:
```suggestion
		log.Printf("Retry send metric: %d\n", i)
```
вроде логгер уже прошли, нужно уже избавляться от fmt в коде. И здесь же стоит залогировать ошибку.